.github/workflows/ci-storybook.yml

```diff
@@ -0,0 +1,38 @@
+name: Build storybooks
+
+on:
+  pull_request
+
+jobs:
+  build_storybooks:
+   runs-on: ubuntu-latest
+   env:
+     APP_ENV: production
+     NODE_ENV: production
+     PANOPTES_ENV: production
```

Comment:
I remember this had to be set to avoid problems with the tests for a couple of components, but I can't remember which ones. Probably also not needed for the storybooks, maybe?